docs-chef-io/content/inspec/resources/k8s_network_policies.md

```diff
@@ -20,61 +20,61 @@ Use the `k8s_network_policies` Chef InSpec audit resource to test the configurat
 ```ruby
 describe k8s_network_policies do
   it { should exist }
-  its('names') { should include 'my-network-policy' }
+  its('NAMES') { should include 'NETWORK-POLICY' }
```

Comment:
`'names'` is not equal to `'NAMES'`. 

When testing as `its('NAMES') { should include 'something' }` it would thrown an error as:
`undefined method 'NAMES' for #<#<Class:0x0000000125e84ee8>:0x000000012096b060>`